lib/data/conversationMessage.ts

```
@@ -369,24 +369,32 @@ export const createConversationMessage = async (
-        messageId: message.id,
-        conversationId: message.conversationId,
-      },
-    });
+    eventPromises.push(
+      inngest.send({
+        name: "conversations/message.created",
+        data: {
+          messageId: message.id,
+          conversationId: message.conversationId,
+        },
+      })
```

ESLint

[error] Insert `,`

(prettier/prettier)